docs/getting-started/gs-environments.md

```diff
@@ -13,10 +13,10 @@ The following table lists all current testing and production environments for EU
 
 All EUID endpoints use the same base URL.
 
-| Environment | Base URL |
-| :--- | :--- |
-| Testing | `https://integ.euid.eu/v2` |
-| Production | `https://prod.euid.eu/v2` |
+| Environment | Cloud Region | Code | Base URL |
+| :--- | :--- | :--- | :--- |
+| Integration Testing | AWS EU West (London) | `eu-west-2` | `https://integ.euid.eu/v2` |
```

Comment:
minor nit: It should really be "AWS Europe (London)" according to AWS [page](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) - not "AWS EU West (London)".
For ap-northeast-1  it is AWS Asia Pacific (Tokyo) and we don't call it Asia Northwest (Tokyo) for example

Reply:
Fixed in follow-up PR, thx.